### PR TITLE
Add extsources to a-b-c packages

### PR DESCRIPTION
**Pull request by ImperatorS79**

### CODE REVIEW

File: packages/b/bzip2/xmake.lua [hunk 1]
```diff
@@ -5,6 +5,14 @@ package("bzip2")
 
     add_urls("https://sourceware.org/pub/bzip2/bzip2-$(version).tar.gz")
     add_versions("1.0.8", "ab5a03176ee106d3f0fa90e381da478ddae405918153cca248e682cd0c4a2269")
+    
+    if is_plat("mingw") and is_subhost("msys") then
+        add_extsources("pacman::bzip2")
+    elseif is_plat("linux") then
+        add_extsources("pacman::bzip2", "apt::libbz2-dev")
```
Comment (SirLynix) [inline packages/b/bzip2/xmake.lua:12]: apt::libbz2-dev isn't right as bzip2 package include binaries

Comment (ImperatorS79) [reply packages/b/bzip2/xmake.lua:12]: Yes, so I need to put both I assume ?

Comment (SirLynix) [reply packages/b/bzip2/xmake.lua:12]: Well it looks like apt/find_package doesn't support binaries anyway, better ask @waruqi

Comment (waruqi) [reply packages/b/bzip2/xmake.lua:12]: > Well it looks like apt/find_package doesn't support binaries anyway, better ask @waruqi

right, only for library.

**Comment by waruqi** [comment]
You can split this patch, submit multiple pr to fix them.

**Pull request closed without merging**